### PR TITLE
Use @oacore/configs

### PR DESCRIPTION
**Pull request by Joozty**
I know it was not necessary but I feel very satisfied now. 😄

### CODE REVIEW

**Comment by viktor-yakubiv** [comment]
Please, update deployment config accordingly before merge.

File: package.json [hunk 1]
```diff
@@ -60,12 +61,13 @@
...
-    "format": "eslint --ignore-path .gitignore --ext .js,.jsx --fix .",
+    "lint": "run-p lint:*",
+    "lint:json": "prettier --ignore-path .gitignore --check '**/*.json'",
+    "lint:js": "eslint --ignore-path .gitignore --ext .js,.jsx .",
     "test-dev": "blc http://localhost:3000 -ro --exclude='/browse' --exclude='/search' --exclude='/public' --exclude='/article-update' --exclude='/api-keys' --exclude='/documentation/api' --exclude='/dataset/getdatadump' --exclude='/recommender/register' --exclude='/images/harvested-by-core-orange.png' --exclude='/images/powered-by-core-orange.png'",
-    "test-local": "yarn test-dev --exclude-external",
+    "test-local": "npm run test-dev -- --exclude-external",
     "test-staging": "blc https://${CORE_STAGING_AREA}core.ac.uk -ro --exclude='/browse' --exclude='/search' --exclude='/public'",
-    "test": "yarn test-dev",
+    "test": "npm run test-dev",
```
[the remark this answers is not in the document]
Comment (Joozty) [reply package.json]: Just because it's one word shorter then `npm run`? 😄

**Comment by Joozty** [comment]
@viktor-yakubiv What do you not like exactly? Like the whole @oacore/configs thing or rules or what?

**Comment by viktor-yakubiv** [comment]
It's just a feeling. I do not know what exactly. Do not get it close, please.

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (viktor-yakubiv) [reply package.json]: Not because of that. I just want to be runner-agnostic if we have the package in dependencies anyway.

**Comment by Joozty** [comment]
That's fine. I thought about yarn -> npm. Unfortunately yarn doesn't support github npm repository. 🙁

File: components/switcher/index.jsx [hunk 2]
```diff
@@ -205,6 +191,25 @@ class Switcher extends Component {
...
+  active: PropTypes.bool,
+  tag: PropTypes.node,
+  actionEvent: PropTypes.oneOf(['click', 'hover']),
+  interval: PropTypes.number,
+  onChange: PropTypes.func,
+  onMouseOver: PropTypes.func,
+  onMouseLeave: PropTypes.func,
+  pauseOnMouseOver: PropTypes.bool,
+}
+Switcher.defaultProps = {
```
[the remark this answers is not in the document]
Comment (Joozty) [reply components/switcher/index.jsx]: https://github.com/oacore/next/pull/203/commits/74d3ecce2481fc731b1e0c6fd97153894e46e0bc

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (Joozty) [reply package.json]: Well you cannot be runner-agnostic. You need to use npm in order  to install @oacore/config anyway.

[the remark this answers is not in the document]
Comment (viktor-yakubiv) [reply package.json]: Fair enough

**Pull request merged**